src/tasks/task1.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+from django.views.generic import CreateView
+from src import models
+
+
+class Task1(CreateView):
+    model = models.Task1Model
+    fields = ('number_of_latent_semantics',)
+    template_name = 'task1.html'
+
+    def get_context_data(self, **kwargs):
```

Comment:
Can you explain what is the purpose of get_context_data and what is context here? Are you going to use it for active page in the front-end?

Reply:
To highlight active tab

Reply:
okay gotcha